setup.py

```diff
@@ -49,7 +49,8 @@
     zip_safe=False,
     platforms='any',
     install_requires=[
-        'Invenio>=1.9999.4',
+        'Invenio>=2.1',
```

Comment:
there is a difference between what "we should" and what "at least works". the PR makes this project workable without my personal Invenio branch, especially when taking the number of my remaining working days and the current (in-)stability of Invenio into account.

the exact requirements needed to be revisited anyway. I hoped to depend on the planned "Invenio-full/core/..." collection package, but as far as I know that one does not exist yet. fiddling around with a very constraint dependency set also does not speed up our development. but especially these considerations are more the ones of @tiborsimko and @pamfilos. here I'm only remove myself out of the equation.

I therefore propose to accept that solution as it is and (as far as I have time) I can implement the "should" solution next week.